models/MITgcm_ocean/model_mod.f90

```diff
@@ -198,6 +227,7 @@ module model_mod
       rkFac, groundAtK1
 
 !--   Input files namelist
+! MEG: These parameters are not used anywhere in the mod_mod!
```

Comment:
if none of the items in this namelist are needed, i'd remove it completely.  the fewer external namelists that need to be tracked the less maintenance is required to track model changes.